README.md

```diff
@@ -49,6 +51,41 @@ This tool allows you to easily build **interactive seat selection** into your we
 </html>
 ```
 
+![Alt text](example.png?raw=true)
+
+## Authentication
+
+### Getting Authenticated
+
+We have a demo user set up, that allows you to create an auth token for our demo so that you can experiment with the seat chart.
+
+#### 1. Get an auth token
+```bash
+curl https://b2b.ingresso.co.uk/api/b2b/ -u "demo:demopass" -ik
+```
+This will have an `x-b2-b-token` in the response header
```

Comment:
Is this header name correct? or should it be `x-b2b-token`?
```suggestion
This will have an `x-b2b-token` in the response header
```

Reply:
OH damn, good catch! 😓 I used VScode's convert to kebab case on `X-B2B-Token` and didn't realise it changed it to that!

Fixed 👉 https://github.com/ingresso-group/feather-demo/commit/5875850eada10a7c01f46b112d4d41b4808d018d